src/service/NylasService.ts

```diff
@@ -137,6 +137,24 @@ export class NylasService extends APIClient {
             }
         });
     }
+
+    /**
+     * This method returns all existing Scheduler pages for the specified nylas account
+     * @param accountId
+     */
+    async getSchedulerPages(accountId: string): Promise<AxiosResponse> {
+        return await this.invokeApi(`/schedule/manage/pages?account_id=${accountId}`, 'GET');
+    }
+
+    /**
+     * This method creates a scheduler page with the given payload. Since this object is really generic there is no good way
+     * to map it into an own class therefore it is an object.
+     * @param payload
+     * @param accountId
+     */
+    async createSchedulerPage(accountId: string, payload: object): Promise<AxiosResponse> {
+        return await this.invokeApi(`/schedule/manage/pages?account_id=${accountId}`, 'POST', payload);
```

Comment:
Wenn du beim invokeApi das Model mit angibst, bekommst du den korrekten Rückgabewert
```suggestion
        return await this.invokeApi<Model>(`/schedule/manage/pages?account_id=${accountId}`, 'POST', payload);
```